examples/image-streams/image-streams-centos7.json

```diff
@@ -14,7 +14,11 @@
         "dockerImageRepository": "openshift/ruby-20-centos7",
```

Comment:
ok.  if we're in agreement that we only want to import explicit tags i'll remove the dockerImageRepository field.